external/fv3fit/fv3fit/train_microphysics.py

```diff
@@ -403,7 +414,7 @@ def _train_function_unbatched(
     if config.shuffle_buffer_size is not None:
         train_ds = train_ds.shuffle(config.shuffle_buffer_size)
 
-    train_set = next(iter(train_ds.batch(50_000)))
+    train_set = next(iter(train_ds.batch(150_000)))
```

Comment:
Not totally intentional since I think this snuck in when doing performance regression checks.  Up from ~5 to ~10 timesteps worth of data.

Reply:
It seems fine since your run is the new best control IMO.